otx/algorithms/common/adapters/mmcv/ops/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Initial file for mmcv ops."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from mmcv.ops import multi_scale_deform_attn
+
+from .multi_scale_deformable_attn_pytorch import multi_scale_deformable_attn_pytorch
+
+multi_scale_deform_attn.multi_scale_deformable_attn_pytorch = multi_scale_deformable_attn_pytorch
```

Comment:
Patching the function in `__init__.py` is not common. How about moving this patching to the `adapters/mmcv/ops/multi_scale_deformable_attn_pytorch.py`